parent/pom.xml

```diff
@@ -86,7 +86,7 @@
         <test.working.dir>${project.build.directory}/testdata</test.working.dir>
         <test.temp.dir>${test.working.dir}/temp</test.temp.dir>
 
-        <starter.version>empty_20230525</starter.version>
+        <starter.version>empty_20240201</starter.version>
```

Comment:
Is this starter going to be used by Integration Tests and Postman Tests as well? If so, shouldn't those other references be updated too?

Reply:
There aren't any other references. The pom.xml from those modules reference this one.